src/lib/Microsoft.Health.Fhir.Ingest/Config/ResourceIdentityServiceType.cs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Ingest.Config
+{
+    /// <summary>
+    /// Resource Identity Service Type.
+    /// </summary>
+    public enum ResourceIdentityServiceType
+    {
+        /// <summary>
+        /// Create the identity resources if not existed.
```

Comment:
nit: do you mind changing this to "Create the identity resources if they do not already exist." or something similar? I am not sure the phrase "not existed" is correct although I did generally understand what it meant

Reply:
Thank you for pointing it out, I'll correct it.